src/smtp/SMTPClient.ts

```diff
@@ -11,7 +12,7 @@ export default class SMTPClient {
 		})
 		sock.on("connect", async () => {
 			console.log("[SMTPClient] Connected to server")
-			sock.write("EHLO localhost\r\n")
+			sock.write("EHLO " + getConfig("host", "localhost") + "\r\n")
```

Comment:
```suggestion
			sock.write(`EHLO ${getConfig("host", "localhost")}\r\n`)
```
use template literals :)